### ISSUE TITLE
[med] : Many pages: Chrome + Jaws: Name & Role is not defined for Back (<) control.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8522]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to any chat window using Tab key and press Enter to select it.
3. Resize the page.
4. Navigate to Back(<) button using Tab key.
5. Observe the screen reader announcement.

## Expected Result:
Name & Role should define for the Back (<) control. Screen reader should read like "Back button".

## Actual Result:
Name & Role is not defined for Back (<) control. Screen reader is reading "unlabeled".

**Note-**
1. On Android screen reader is 'Silent'
2. On MAC screen reader is 'group'
 
**Other occurrences**
>Same issue repro on PR #8795, #8969

## Workaround:
Yes

## Area issue was found in:
All pages

## Failed WCAG checkpoints
4.1.2

## User impact:
Name & Role are the fundamental attribute for accessibility. If the role & name are not announced, then Screen reader user will have difficulty understanding the purpose of the button.

## Suggested resolution:
Make sure that screen reader announces the correct name of the button.           

Refer to:    https://knowbility.org/blog/2021/whats-in-a-name

## Platform:
- Web
- Android
- Desktop App
- Mobile Web

**Version Number: ** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti 

https://user-images.githubusercontent.com/97600950/162132787-55fc8380-e78f-4ac0-9b95-8ef6e309adb2.mp4
